test/tests/concrete_ASR_validation/asr_validation.i

```diff
@@ -71,7 +71,7 @@
     strain = SMALL
     add_variables = true
     eigenstrain_names = 'asr_expansion'
-    generate_output = 'stress_xx stress_yy stress_zz stress_xy stress_yz stress_zx vonmises_stress hydrostatic_stress elastic_strain_xx elastic_strain_yy elastic_strain_zz strain_xx strain_yy strain_zz '
```

Comment:
I'm a little confused here -- the output of several stress components is disabled, but they are still in the .csv file. I'm guessing that the .csv files checked in were actually generated before these components were removed.

I think it is valuable to check the stress components, even if they are close to zero. There is an option to set tolerances specific to each variables with csvdiff. Look at the `override_columns`, `override_rel_err` and `override_abs_zero` options. You ought to be able to use those to set a loose abs_zero (maybe 1e-5) just for the stresses (which are the ones that are problematic because they are so low that relative error checks don't work), but keep a tight tolerance for everything else.

Reply:
The tests are failing in `Exodiff` either in mac or in civet. What is the equivalent option for `override_rel_err` in `Exodiff`? Should I just remove the `Exodiff` tests and stick with `CSVDiff` only?

Reply:
Either way works. To fine-tune exodiff tests, you can create a custom comparison file. Here's the process:
1) run exodiff --summary out.e > myfile.cmp
2) You can add custom relative or absolute tolerances on variables right after the variable names. Look for .cmp files in the MOOSE repo for examples.
3) Check in that .cmp file and add a line in the test block that points to it. I think it's 'custom_cmp='

Reply:
Since the tests are passing, we can skip this for now.

Reply:
So do the updated gold exodus files just have these variables removed?

Reply:
I added back the stresses and using a 'custom_cmp' now. It's ready.